apps/address-service/README.md

```diff
@@ -1,3 +1,21 @@
+# Description
+It's a separate service for address processing.
```

Comment:
It's still not clear what it does without digging into code and what the difference between `/suggest` and `/search` API methods. Please describe for a person, that sees this first time and not read the code.

What I'm understood (correct me if something is wrong):

```suggestion
Returns detailed information about property by address search string. Uses remote data sources and local. Remote data sources are official opened APIs. Local data source is used in case when required property is not found by remote data sources or it needs to be corrected.
```